test/specs/PushManager.test.ts

```diff
@@ -113,6 +114,12 @@ describe(PushManager, () => {
       await pushManager.isWebPushSubscribed()
 
       expect(createRequestSpy).toHaveBeenCalledTimes(2)
+      expect(stringifySpy).toHaveBeenCalledTimes(2)
+      expect(stringifySpy).toHaveBeenLastCalledWith({
```

Comment:
I'd suggest checking the createRequest parameters instead of the `JSON.stringify` calls. This makes sure that the parameter is sent to `createRequest` properly, instead of the following:

```ts
// passes the test, but functionality is broken
const json = JSON.stringify({ ... });
console.log(json); // var used
createRequest() // without passing json
```

It would also hide the implementation detail on how exactly JSON is stringified.

Reply:
Thanks, updated as suggested!